tests/symmetry_tests.py

```diff
@@ -0,0 +1,206 @@
+                        'A     A    B1   B2   B3   \n'
+                        'B1    B1   A    B3   B2   \n'
+                        'B2    B2   B3   A    B1   \n'
+                        'B3    B3   B2   B1   A    \n')
+
+        c2h_char_tbl = ('==========> C2h <==========\n\n'
+                        '      Ag   Bg   Au   Bu   \n\n'
+                        'Ag    Ag   Bg   Au   Bu   \n'
+                        'Bg    Bg   Ag   Bu   Au   \n'
+                        'Au    Au   Bu   Ag   Bg   \n'
+                        'Bu    Bu   Au   Bg   Ag   \n')
+
+        c2v_char_tbl = ('==========> C2v <==========\n\n'
+                        '      A1   A2   B1   B2   \n\n'
+                        'A1    A1   A2   B1   B2   \n'
+                        'A2    A2   A1   B2   B1   \n'
+                        'B1    B1   B2   A1   A2   \n'
+                        'B2    B2   B1   A2   A1   \n')
+
+        d2h_char_tbl = ('==========> D2h <==========\n\n'
+                        '      Ag   B1g  B2g  B3g  Au   B1u  B2u  B3u  \n\n'
+                        'Ag    Ag   B1g  B2g  B3g  Au   B1u  B2u  B3u  \n'
+                        'B1g   B1g  Ag   B3g  B2g  B1u  Au   B3u  B2u  \n'
+                        'B2g   B2g  B3g  Ag   B1g  B2u  B3u  Au   B1u  \n'
+                        'B3g   B3g  B2g  B1g  Ag   B3u  B2u  B1u  Au   \n'
+                        'Au    Au   B1u  B2u  B3u  Ag   B1g  B2g  B3g  \n'
+                        'B1u   B1u  Au   B3u  B2u  B1g  Ag   B3g  B2g  \n'
+                        'B2u   B2u  B3u  Au   B1u  B2g  B3g  Ag   B1g  \n'
+                        'B3u   B3u  B2u  B1u  Au   B3g  B2g  B1g  Ag   \n')
+
+        char_tables = [c1_char_tbl,
+                       c2_char_tbl,
+                       cs_char_tbl,
+                       ci_char_tbl,
+                       d2_char_tbl,
+                       c2h_char_tbl,
+                       c2v_char_tbl,
+                       d2h_char_tbl]
+
+        for count, group in enumerate(groups):
+
+            mol = system_factory(system_type = 'molecule',
+                                             build_type = 'psi4',
+                                             basis='sto-3g',
+                                             mol_geometry = [('O', (0., 0., 0)),
+                                                             ('H', (0., 0, -1.5)),
+                                                             ('H', (0., 0, 1.5))],
+                                             symmetry = group)
+
+            self.assertEqual(mol.point_group, [group, irreps[count]])
+            self.assertEqual(mol.orb_irreps, orb_irreps[count])
+            self.assertEqual(mol.orb_irreps_to_int, orb_irreps_to_int[count])
+
+            with patch('sys.stdout', new = StringIO()) as fake_out:
+                char_table([group, irreps[count]])
+                self.assertEqual(fake_out.getvalue(), char_tables[count])
+
+    def test_symmetry_uccnvqe(self):
```

Comment:
It's clearer to combine this test and the two below into a single test. Please use the `param` feature of `pytest`. Example [here](https://github.com/psi4/psi4/blob/master/tests/pytests/test_dipoles.py).